app/lib/esClient.js

```diff
@@ -1,22 +1,38 @@
 const log = require('./logger');
 const elasticsearch = require('elasticsearch');
 const esConfig = require('../../config/config').es;
-const queryBuilder = require('./queryBuilder');
+const buildNearestQuery = require('./queryBuilder').buildNearestQuery;
+const buildNearestOpenQuery = require('./queryBuilder').buildNearestOpenQuery;
 
 const client = elasticsearch.Client({ host: `${esConfig.host}:${esConfig.port}` });
 
 function mapResults(results) {
   return results.hits.hits.map((hit) => {
     // eslint-disable-next-line no-underscore-dangle
     const pharmacy = hit._source;
-    pharmacy.dis = hit.sort[0];
+    pharmacy.distanceInMiles = hit.sort[0];
     return pharmacy;
   });
 }
 
 async function getPharmacies(location, radius, size) {
   try {
-    const results = await client.search(queryBuilder(location, radius, size));
+    const results = await client.search(buildNearestQuery(location, radius, size));
+    log.info({
+      numberOfResults: results.hits.total, location, radius, size
+    }, 'ES results returned.');
```

Comment:
Is it worth including the fact these results are returned from the nearby query?